File: C Language/pro69.c
/**
    bitwise and operator
*/

#include<stdio.h>

void main(){
    int ip1, ip2, ans;

    ip1 = 13;
    ip2 = 10;

    /**
        ip1     13      1101
        ip2     10      1010
                &       1000
    */
    ans = ip1 & ip2;

    printf("\n Answer is %d", ans);
}

File: NodeJS/pro69.js
/*
    function return value
*/

function makeSum(ip1 = 0, ip2 = 0, ip3 = 0){
    return ip1 + ip2 + ip3;
}

let ans = makeSum(11);
console.log("Sum is",ans);
ans = makeSum(11, 22);
console.log("Sum is",ans);
ans = makeSum(11, 22, 33);
console.log("Sum is",ans);
ans = makeSum();
console.log("Sum is",ans);